discovery-provider/docker-compose.yml

```diff
@@ -81,6 +81,38 @@ services:
       - override.env
     environment:
       - audius_discprov_infra_setup=audius-docker-compose
+      - audius_no_workers=false
+      - audius_db_run_migrations=false
+    networks:
+      - discovery-provider-network
+
+  indexer:
+    image: audius/discovery-provider:${TAG:-c54dc44721ad3c8352f0cec56b94002cffe9512a}
```

Comment:
Let's use `container_name: indexer` so that ELK can grab logs.

Reply:
Changes have already been made, built, and pushed to Docker Hub via this commit for the Filebeat container:

* https://github.com/AudiusProject/audius-protocol/pull/3846/commits/c3a1c526e62b482c3be12db8d818de989fabfe5e